src/components/operations/operation-details/ko/runtime/operation-details.html

```diff
@@ -326,21 +326,19 @@ <h3>Protocol</h3>
 <!-- ko if: apiType() === 'graphql' -->
 <div class="animation-fade-in">
     <div class="operation-header">
-        <h2 class="operation-name"></h2>
+        <div>
+            <h2 class="gql-details">GraphQL Endpoint</h2>
```

Comment:
The class `.gql-details` overrides the sizes defined by users. We should do it carefully or don't do it at all (also, we plan to add local styling to complex components, so that users will be able to adjust those as they like).